crates/turbopack-ecmascript/src/side_effect_optimization/locals/chunk_item.rs

```diff
@@ -100,4 +100,13 @@ impl ChunkItem for EcmascriptModuleLocalsChunkItem {
     fn module(&self) -> Vc<Box<dyn Module>> {
         Vc::upcast(self.module)
     }
+
+    #[turbo_tasks::function]
+    async fn is_self_async(&self) -> Result<Vc<bool>> {
+        if let Some(async_module) = *self.module.get_async_module().await? {
+            Ok(Vc::cell(*async_module.is_self_async().await?))
```

Comment:
Is there a reason for dropping connection here?

Reply:
What do you mean? which connection is dropped?

Reply:
I meant the creation of Vc<bool> with an awaited value. IIRC, it will make this whole function execute again when the value is changed, right?

Reply:
ah yes